### PR TITLE
moved all the tools to yml

### PR DESCRIPTION
**Pull request by alexlenail**

### CODE REVIEW

File: galaxyp.yaml [hunk 1]
```diff
@@ -178,3 +178,124 @@ tools:
...
+
+- name: openms
+  owner: galaxyp
+  tool_panel_section_label: OpenMS
+  tool_shed_url: http://testtoolshed.g2.bx.psu.edu/
+
+- name: spectrast
+  owner: iracooke
+  tool_panel_section_label: SpectraST
+  tool_shed_url: http://testtoolshed.g2.bx.psu.edu/
```
Comment (bgruening) [inline galaxyp.yaml:285]: I guess most of the testtoolshed tools are nowadays already in the MTS. I will check this in the next days.

**Comment by bgruening** [comment]
Don't worry about the failing tests. I think the build time was way to long. The mid-term plan is to move more and more tools over to conda dependencies and make installation faster with this.

Thanks!

**Comment by alexlenail** [comment]
I'll look over the "testtoolshed" stuff. 

Unrelated to this PR: Where can I get more information about the change over to conda? I still don't really understand what it will entail.

**Comment by alexlenail** [comment]
@bgruening looks like nearly all the tools that are marked as "testtoolshed" are still in the testtoolshed, not the main tool shed. The only one which needed changing was https://github.com/bgruening/docker-galaxyp/pull/5/commits/3b6f7e4f699093c7064fa5ad48335ddad97b777c

**Comment by alexlenail** [comment]
(perhaps those tools should be moved/copied to the main tool shed? This PR is good to go though)

**Comment by bgruening** [comment]
@zfrenchee you are right! We just need more time for galaxyp. It's on my todo list :(
Regarding conda packages, it's a new way of defining tool dependencies in Galaxy. And this way allows you to install pre-compiled packages, in contrast to the former method of choice, which was compiling everything from source. So this will speed up installation dramatically ... but we need to create such conda packages before.

**Comment by alexlenail** [comment]
@bgruening can you show me an example of a conda-package tool being imported/used in galaxy?  I understand the theory, but I don't know how it looks in practice.

**Comment by bgruening** [comment]
This one is a Conda enabled Docker container: https://github.com/gianlucacorrado/galaxy-RNAcommender/blob/master/Dockerfile#L7

Galaxy will automatically fetch packages from bioconda and installs them under _tool_deps_/_conda.

**Comment by alexlenail** [comment]
So that Dockerfile doesn't install any packages but _allows_ for the admin to install from conda instead of the regular toolshed?

**Comment by bgruening** [comment]
Yes. More or less. Note that this only hold true for dependencies. Tools are still installed, so the XML file. Also you can decide if you want to install the binaries during install time or during tool-run time. Even the first case is way faster than compiling on your own, because you don't need to compile.

**Pull request merged**